docs/specs/api/baseline-api-v1.0-psd01.md

```diff
@@ -208,17 +226,208 @@ This interface provides functions to manage vaults and keys.
 | VAULT11| createVaultSecret(vaultId: string, params: object): Promise<any>; |
 | VAULT12| deleteVaultSecret(vaultId: string, secretId: string): Promise<any>; |
 
+## 3.4 Agreement Execution
+
+
+
+### 3.4.1 IBaselineRPC
```

Comment:
@Consianimis there are some updates to IBaselineRPC that are described in this PR https://github.com/ethereum-oasis/baseline/pull/259

Reply:
@bitwiseguy, I have updated as discussed yesterday.